target/linux/ar71xx/image/generic.mk

```diff
@@ -205,6 +244,28 @@ define Device/dr531
 endef
 TARGET_DEVICES += dr531
 
+define Device/om2p
+  $(Device/OpenMesh)
+  DEVICE_TITLE := OpenMesh OM2P/OM2P-HS/OM2P-LC
+  BOARD_NAME = OM2P
+endef
+TARGET_DEVICES += om2p
+
+define Device/om5p
+  $(Device/OpenMesh)
+  DEVICE_TITLE := OpenMesh OM5P/OM5P-AN
+  BOARD_NAME = OM5P
+endef
+TARGET_DEVICES += om5p
+
+define Device/om5pac
+  $(Device/OpenMesh)
+  DEVICE_TITLE := OpenMesh OM5P-AC
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x om-watchdog
```

Comment:
Not a typo. The complete line has to be changed to remove the om-watchdog and then change it to +=